mkdocs.yml

```diff
@@ -95,7 +95,6 @@ nav:
     - API Design: guides/api-design.md
     - CRD Management: guides/crd-management.md
     - Implementer's Guide: guides/implementers.md
-    - Documentation Style Guide: guides/documentation.md
```

Comment:
Note: I'm intentionally not adding a redirect here because the page was _just_ added and I can't imagine that anything has had a chance to link to it. We also do not have any other internal links to the page.